internal/test/wmcb/wmcb_test.go

```diff
@@ -412,4 +412,14 @@ func testWMCBCluster(t *testing.T) {
 		metav1.ListOptions{LabelSelector: e2ef.WindowsLabel})
 	require.NoErrorf(t, err, "error while getting Windows node: %v", err)
 	assert.Lenf(t, winNodes.Items, 1, "expected one node to have node label but found: %v", len(winNodes.Items))
+	// Test Windows Nodes for Ready status
+	for _, node := range winNodes.Items {
+		for _, condition := range node.Status.Conditions {
+			if condition.Type == v1.NodeReady {
```

Comment:
Won't `condition.Type` not being present result in a false positive?

Reply:
Agreed, the NodeReady label being missing should cause a test failure